src/containers/ConnectorManagement.jsx

```diff
@@ -106,6 +109,14 @@ export default class ConnectorManagement extends Component {
     this.gotoParent()
   }
 
+  handleTimeout () {
+    this.setState({ jobInBackground: true })
+  }
+
+  accountConfig () {
+    this.setState({ jobInBackground: false })
```

Comment:
Yep, it works. 

It did not get your point about `isConnected`. That state value is already true at that point.